domains/herkura.json

```diff
@@ -0,0 +1,10 @@
+{
+  "description": "Herkura's Portfolio",
+  "repo": "https://github.com/herkura/.me",
+  "owner": {
+    "username": "herkura",
```

Comment:
We need a way to contact you in case you need to be notified of any changes to your domain or the is-a-dev service. Alternatively, you can keep email as "" and add an active account username for twitter, discord, etc.